src/Detector/Adapter/DrupalAdapter.php

```diff
@@ -131,15 +131,18 @@ public function detectModules(\SplFileInfo $path)
         foreach ($files as $file) {
             $contents = file_get_contents($file->uri);
 
-            if (preg_match('/project' . $seperator . ' (.+)/', $contents, $matches) &&
-              preg_match('/version' . $seperator . '(.+' . $this->majorVersion . '\.x-.+)/', $contents, $verMatches)
+            if (preg_match_all('/project' . $seperator . ' (.+)/', $contents, $matches) &&
```

Comment:
is it expected to have a space after the seperator? `$seperator . ' (.+)/'` vs `$seperator . '(.+)/'`